specification/consumption/resource-manager/readme.md

```diff
@@ -57,24 +65,3 @@ csharp:
   output-folder: $(csharp-sdks-folder)/Consumption/Management.Consumption/Generated
   clear-output-folder: true
 ```
-

```

Comment:
Is there a reason to remove Go from the list?

Reply:
No reason, it was just missed while copying over from the private repo. Adding it back